galasa-ui/src/tests/footer.test.tsx

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Move this file into the `tests/components` folder to keep the component tests organised?

Reply:
Also really minor but it would be nice to rename the file to Footer.test.tsx, just to match up with the component in Footer.tsx (again to keep consistency with other component tests)